input/fsh/actors.fsh

```diff
@@ -19,7 +19,7 @@ Usage: #definition
 * name = "AUeRequestingFiller"
 * title = "AU eRequesting Filler"
 * status = #draft
-* description = "The AU eRequesting Filler is a system that is responsible for finding and retrieving diagnostic service requests so that service providers can fulfil them. It ensures that service providers have the necessary information to fulfil the requests and manages the requests through the fulfilment process."
+* description = "The AU eRequesting Filler is a system that is responsible for finding and retrieving diagnostic request fulfilments so that service providers can fulfil them. It ensures that service providers have the necessary information to fulfil the requests and manages the requests through the fulfilment process."
```

Comment:
I don't think this implements [FHIR-46801](https://jira.hl7.org/browse/FHIR-46801) resolution to replace the description with the text in the Resolution Description.
Please check with @michael-wilson-au.

Reply:
[FHIR-46801](https://jira.hl7.org/browse/FHIR-46801) moved to PR covering consolidation of actor definitions.